src/app/core/views/portinfo/digital/index.ts

```diff
@@ -28,6 +28,12 @@ export const DigitalPortInfo: PortInfoRecord<DigitalComponent> = {
             "outputs": [{ origin: V(0.62, 0), target: V(1.32, 0), dir: V(+1, 0) }],
         }],
     },
+    "ConstantNumber": {
+        ...DefaultDigitalPortInfo,
+        PositionConfigs: [{
+            "outputs": CalcPortPositions(4, 0.507 - DEFAULT_BORDER_WIDTH/2, 1, V(1, 0)),
```

Comment:
Generally discourage "magic" numbers, where did the `0.07` come from?

Reply:
I used the 0.07 because the corners looked the best when zoomed in. Instead I changed it to `.5` from `.507 - DEFAULT_BORDER_WIDTH/2`, but the top output port and the top of the line connecting outputs is the smallest bit misaligned. Not sure where to find actual numbers in original code, let me know what you think of the change when I commit it.